test/WebSites/ResponseCacheWebSite/Controllers/CacheProfilesController.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Mvc;
+
+namespace ResponseCacheWebSite.Controllers
+{
+    public class CacheProfilesController
```

Comment:
nit: I think @yishaigalatzer wanted funcional websites to only use attribute routing in general.